tests/zfs-tests/cmd/mmapwrite/mmapwrite.c

```diff
@@ -67,7 +67,7 @@ int
 main(int argc, char **argv)
 {
 	int fd;
-	char buf[BUFSIZ];
+	char buf[BUFSIZ] = {0};
```

Comment:
Using BUFSIZ here probably isn't a great idea, even if that is what the existing code does.  How about we just make this 1024 and initialize it with `memset()` with a fixed character pattern.  This should be fine since the intent in to overlap the IO with the 8192 mmaps region.

```
        char buf[1024];

        memset(buf, 'a', sizeof (buf));
```